evm/vm/forks/sharding/validation.py

```diff
@@ -0,0 +1,31 @@
+from evm.exceptions import (
+    ValidationError,
+)
+from evm.validation import (
+    validate_transaction_access_list,
+)
+
+
+def validate_sharding_transaction(vm, transaction):
+    # `validate_homestead_transaction` assumes underlying signature scheme
+    # to be ECDSA and checks signatures accordingly.
+    # `validate_frontier_transaction` checks for the balance of transaction
+    # sender.
+    # Both of which are in confilct with Account Abstraction implementation.
```

Comment:
I think this whole comment can be removed with the understanding that this is just a new validation function.